tools/ReleaseHelpers.cs

```diff
@@ -10,19 +9,14 @@ public static class ReleaseHelpers
     {
         public static ICollection<int> GetIssueNumbersReferencedFromReleases(IEnumerable<Release> releases)
         {
-            // Release bodies should contain references to fixed issues in the form
-            // (#1234), or (#1234, #1235, #1236) if multiple issues apply to a topic.
-            // It's hard (impossible?) to harvest values from a repeated capture group,
-            // so grab everything between the ()s and split manually.
             var issuesReferencedFromRelease = new HashSet<int>();
             foreach (var release in releases)
             {
-                foreach (Match match in Regex.Matches(release.Body, @"\((?<issueNumbers>#[0-9]+((, )#[0-9]+)*)\)"))
+                foreach (Match match in Regex.Matches(release.Body, @"\(\s*#(?<issueNumber>[0-9]+)(,\s*#(?<issueNumber>[0-9]+))*\s*\)"))
                 {
-                    var issueNumbers = match.Groups["issueNumbers"].Value;
-                    foreach (var issueNumber in issueNumbers.Split(new[] { '#', ' ', ',' }, StringSplitOptions.RemoveEmptyEntries))
+                    foreach (Capture capture in match.Groups["issueNumber"].Captures)

```

Comment:
AH!!! 😻

Reply:
I thought you'd like it! Somehow I had missed this comment when you first wrote it.